scripts/install/install-linux-arm64.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
Instead of duplicating scripts for each os+arch, since I think the only difference is the arch name, I'd like for us to look at using `go env GOARCH` to determine the arch value and then use that as a variable. Then we just have to maintain 3 install scripts.